test/upgrade/installation/post_install.go

```diff
@@ -0,0 +1,34 @@
+/*
+Copyright 2020 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package installation
+
+import pkgupgrade "knative.dev/pkg/test/upgrade"
+
+const (
+	postInstallShellFuncName = "apply_post_install"
+)
+
+func PostInstall() pkgupgrade.Operation {
+	return pkgupgrade.NewOperation("EventingPostInstall", func(c pkgupgrade.Context) {
+		c.Log.Info("Running shell function: ", postInstallShellFuncName)
+		err := callShellFunction(postInstallShellFuncName)
+		if err != nil {
+			c.T.Error(err)
+			return
+		}
```

Comment:
You probably don't need all the trouble to add a shell function, couldn't you just add `apply_post_install` after: https://github.com/knative/eventing/blob/1e06e5ce4c64d0f31e7f687fff942493e35fe060/test/upgrade/installation/git_head.go#L29

Reply:
I did this in this way because they are 2 separate operations/manifests and looking at `TestEventingUpgrades` it's more explicit what are the operations required for an upgrade: install, and then post-install.
At the end of the day, this is just a sequence of steps. 
WDYT?

Reply:
Actually do you even need to change this at all? The upgrade test currently does not test the sources at all. So it shouldn't fail the upgrade test.

But if the question is to extend the upgrade test to test all the resources, including the api server source, maybe we can have the discussion with @cardil and do it in another PR.

Reply:
I'll remove upgrade test changes then.

Reply:
I removed test changes, we can figure out later to extend the upgrade test to test all sources.